src/MainUtils.cpp

```diff
@@ -288,9 +288,11 @@ string getTargetCpuOption() {
 
 string getTargetTripleOption() {
   string targetOptions = "";
+  // Comand cannot tolerate extra spaces. Add only when needed.
   if (mtriple != "")
     targetOptions = "--mtriple=" + mtriple;
-  // Comand cannot tolerate extra spaces. Add only when needed.
+  if (mtriple == "" && kHostTriple != "")
```

Comment:
NIT:
Isn't that just a 'else if (kHostTriple != "")`

Reply:
Oh, yeah. I had somewhat more complicated logic when I was originally working on options for the change and missed this during the clean up.